extensions/binding/org.eclipse.smarthome.binding.homematic/README.md

```diff
@@ -211,18 +212,29 @@ The receiveDelay is handy for dimmers and rollershutters for example.
 If you have a slider in a UI and you move this slider to a new position, it jumps around because the gateway sends multiple events with different positions until the final has been reached.
 If you set the ```receiveDelay``` to some seconds, these events are filtered out and only the last position is distributed to the binding.
 The disadvantage is of course, that all events for this channel are delayed.
+*The channel configs are optional*
 
 ```java
   Thing HM-LC-Dim1T-Pl-2    JEQ0999999 "Name"  @  "Location" {
       Channels:
-          Type HM-LC-Dim1T-Pl-2_1_level : 1#LEVEL [
+          Type HM-LC-Dim1T-Pl-2_1_LEVEL : 1#LEVEL [
```

Comment:
As far as I see this on the code, it is important to use the correct case (because it is compared case sensitive).
However, since the channels are actually generated at runtime, I do not know if the case is different depending on which type of gateway you are using.